src/views/terms-and-conditions.html

```diff
@@ -66,7 +66,7 @@
                     Disruptive or Unethical Conduct
                 </h4>
                 <p>
-                    You acknowledge and agree that the Organisers and other security personnel, volunteers at the Event, or staff of Cambridge Live reserve the right to remove you from the Event if the Organisers, in their sole discretion, determine that your presence or behavior create a disruption or hinder the Event or the enjoyment of the Event by other attendees or to be of an unethical or harrassing nature. All wristbands or distributed items (including swag from sponsors) must then be returned to the Organisers upon request.<br>You also agree to abide by the <br><a href="http://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank">MLH Code of Conduct</a>
+                    You acknowledge and agree that the Organisers and other security personnel, volunteers at the Event, or staff of Cambridge Live reserve the right to remove you from the Event if the Organisers, in their sole discretion, determine that your presence or behavior create a disruption or hinder the Event or the enjoyment of the Event by other attendees or to be of an unethical or harrassing nature. All wristbands or distributed items (including swag from sponsors) must then be returned to the Organisers upon request.<br>You also agree to abide by the <br><a href="http://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank">MLH Code of Conduct.</a>
```

Comment:
I'm not convinced that the full stop should be part of the link, since the sentence that "MLH Code of Conduct" is part of is not all linked.  However, this illustrates a problem with the arrow, since if we move the full stop after the link, there will be a large space between the text and the full stop where the arrow should appear.  I'm wondering how we'd deal with links in the middle of a sentence — do we simply omit the arrow?  Perhaps it is best to treat this situation like a link in the middle of a sentence too, since the whole sentence is not part of the link.

Also, I've just noticed that adding the new line in the middle of the sentence (before "MLH Code of Conduct") looks odd on small screens, where the sentence breaks in the middle of the line unexpectedly.

Reply:
Indeed, this is why I didn't put the full stop in to start off with. It might be better to have "…by the MLH code of conduct. Read the MLH code of conduct →" The line break was for desktops but thinking about it, this is a bad use of a line break (when producing non-responsive designs it's common to break the lines manually to improve readability, but HTML is far from non-responsive). So I would suggest removing the break and adding a second sentence.